Modules/Server/Sources/PocketCastsServer/Public/Refresh/RefreshManager.swift

```diff
@@ -30,6 +30,22 @@ public class RefreshManager {
         refreshQueue.addOperation(UpNextSyncTask())
     }
 
+
+    /// Updates a podcast and all the associated information.
+    ///
+    /// Note that this will force all the episodes to be updated.
+    /// - Parameter podcast: a `Podcast` object
+    public func refresh(podcast: Podcast) {
+        podcast.forceRefreshEpisodes = true
+        refresh(podcasts: [podcast]) {
+            if SyncManager.isUserLoggedIn() {
+                guard let episodes = ApiServerHandler.shared.retrieveEpisodeTaskSynchronouusly(podcastUuid: podcast.uuid) else { return }
```

Comment:
We update the episode info because the user might have listened to some episodes on another platform.

If we don't do that we risk adding the episodes without their data.